src/app/services/core.service.ts

```diff
@@ -25,17 +25,24 @@ export interface HomeySettings {
 export interface GlobalSettings {
   homey: HomeySettings;
   darkSky: DarkSkySettings;
+  panel: PanelSettings;
 }
 
 /* Panel Settings */
-export interface PanelSettings {
+export interface CommonSettings {
   icon: string;
+  show: boolean;
```

Comment:
It's a detail, but isn't visible a better name since it's a noun?

Reply:
totally agree, will change it tomorrow morning

Reply:
Done! Would you change the `settings.panel.showHeader` as well?

Reply:
headerVisible maybe?